redaxo/src/addons/install/lib/package/package_download.php

```diff
@@ -84,6 +84,6 @@ private function isCorrectFormat($file)
             return $success;
         }
 
-        return file_exists("phar://$file/" . $this->addonkey);
+        return is_file("phar://$file/" . $this->addonkey);
```

Comment:
```suggestion
        return is_dir("phar://$file/" . $this->addonkey);
```

Reply:
Ist das hier nicht ein zip file?

Reply:
ist das net ein ordner in einem zip file?

Reply:
da kommt sowas raus wie `phar://path/to/addon.zip/addon`.
Also wie @bloep sagt, ein Ordner in einer Zip-Datei.
Somit müsste es aber denke ich `is_dir` sein.